tests/osm/test_validate_osm.py

```diff
@@ -0,0 +1,506 @@
+"""Test validate_osm."""
```

Comment:
Is it possible to add a small test for `_compile_tags`?. While this is a very small function, it may be useful to have.